theme/templates/resource-landing-page/citation.html

```diff
@@ -155,6 +157,7 @@
                               action="/hsapi/_internal/{{ cm.short_id }}/set-resource-flag/"
                               method="POST">
                             <input name="t" type="hidden" value="make_private"/>
+                            <input name="resource-mode" type="hidden" value="{{ resource_mode }}">
```

Comment:
@pkdash can these ```resource-mode``` hidden input field statements in multiple places be consolidated into one statement? In other words, will one hidden ```resource-mode``` hidden input field be sufficient to be used in different condition cases?